PICMI_Python/applied_fields.py

```diff
@@ -167,3 +167,24 @@ def __init__(self, x_front_location=None, y_front_location=None, z_front_locatio
 
         self.handle_init(kw)
 
+        
+class PICMI_LoadGriddedField(_ClassWithInit):
+    ''' "The data read in is used to initialize the E and B fields on the grid at the start of the simulation. 
```

Comment:
This looks good, thanks!
Though sorry for my pickiness, for consistency with the other classes, can you use double quotes here? Also, remove the extra space and double quote at the start of the comment.

Reply:
No worries, thanks for being thorough! Fixed